src/ui/index.js

```diff
@@ -9,7 +9,7 @@ const customTracker = require('../utils/customTracker')
 const signale = require('../utils/signale')
 
 const index = () => {
-	return layout('<div id="main"></div>', 'favicon.ico', [ 'index.css' ], [ 'index.js' ], {
+	return layout('<div id="main"></div>', `${ config.baseUrl }/favicon.ico`, [ `${ config.baseUrl }/index.css` ], [ `${ config.baseUrl }/index.js` ], {
```

Comment:
Wouldn't it be possible to remove the `/` (and config.baseUrl) at all and just use `index.css` and `index.js`?

Reply:
This might work with the api path, too. I'm not sure if the `/` is required at all. Could removing it avoid the new option?